cmd/cli.go

```diff
@@ -56,7 +48,8 @@ func main() {
 	// Update mechanism
 	update := make(chan struct{}, 1)
 	if len(args) > 0 && args[0] == "version" {
-		update <- struct{}{}
+		fmt.Println(command.Version)
```

Comment:
This variable gets set during the build process `scripts/compile.sh:40`. At this point we don't have to ask the backend, the current build version is known here.